docs/aws-tutorial.md

```diff
@@ -0,0 +1,283 @@
+# Hosting AzerothCore via Amazon AWS
+
+​	Preface: The goal with this guide is for it to be mostly via a command line. There is also the [AWS Console](https://aws.amazon.com/console/) that some will find more intuitive, however, for guide longevity (as the AWS Console UI may change) the aws-cli tool will be recommended. Also included will be grep commands to receive only the required output. If the entire output is desired simply remove the `| grep` and everything after it.
```

Comment:
This is just minor stuff, but I'd rather see the jump-in before Preface to be removed since it only added a small normal space which makes it look like a mistake and instead of intended :P 
![image](https://user-images.githubusercontent.com/24550914/59166824-e4cdea00-8b2d-11e9-9791-ee37f2f2cd8d.png)

Reply:
Solved ;-)